src/service/status.service.ts

```diff
@@ -0,0 +1,57 @@
+import { User } from 'models';
+import Product from '../models/products.Model';
+import Vendor from '../models/vendor.model';
+
+export class statusService {
```

Comment:
You can't create a whole new service just for this. Move these to the product service